rj_gameplay/rj_gameplay/play/penalty_offense.py

```diff
@@ -33,8 +33,8 @@ def tick(
         if self._state == State.INIT:
             self.prioritized_tactics.append(striker_tactic.StrikerTactic(world_state))
             num_liners = len(world_state.our_visible_robots) - 1
-            start_pt = np.array([-2.0, 1.5])
-            end_pt = np.array([2.0, 1.5])
+            start_pt = np.array([-2.4, 1.5])
+            end_pt = np.array([2.4, 1.5])
```

Comment:
Is there a particular reason why you widened the length of the line that the robots LineUp at?

Reply:
The only reason for that was to more evenly spread out the bots on the field but I guess it makes more sense for them to be bunched up tactically.

Reply:
I think I'll keep the robots more spread out in case the penalty is saved and moves towards the edges of the field.